README.md

```diff
@@ -100,6 +100,8 @@ will be queried from the triplestore and translated accordingly.
 The service also allows for configuration by means of the following environment
 variables:
 
+* **VIRTUOSO_ENDPOINT**: *(default: `http://virtuoso:8890/sparql`)* the direct virtuoso endpoint
```

Comment:
perhaps recall it HIGH_LOAD_DATABASE_ENDPOINT and default it to http://database to not break any assumptions.